### PR TITLE
 #1177 Updated the Heroku Deploy docs according to the new announcement

### PR DESCRIPTION
**Pull request by prokan468**
-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

[skip ci]

### CODE REVIEW

File: docs/Features/heroku.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This sub-generator allows the deployment of your JHipster .Net application to Heroku cloud.
 
-The Heroku sub-generator will always use free tiers/options. Nevertheless installing addons needs a properly verified Heroku account. 
+The Heroku sub-generator will always use free tiers/options but there would be no resources and dynos available hence a paid plan of Heroku is required.
```
Comment (ivanmonteiro) [inline docs/Features/heroku.md:5]: Please remove the "will always use free tiers/options" part

File: docs/Features/heroku.md [hunk 2]
```diff
@@ -33,9 +33,9 @@ jhipster heroku
 - Postgres ✔ 
 - MSSQL ✔ (requires a manual step described below)
 
-Currently MySql and PostgreSQL database addons are free and automatically added by this sub-generator when deploying to Heroku. Even though those resouces are free, an accound with verified credit card is required to add them.
+The MySql and PostgreSQL database addons are not free of cost and won't be automatically added by the sub-generator when deploying to Heroku. A paid plan is required to use the resources and addons.
```
Comment (ivanmonteiro) [inline docs/Features/heroku.md:36]: Some db addons are still free. Other addons (specially those managed by heroku itself) will be paid. The generator will deploy them anyway (except the MSSQL one for now)

So this line can be changed to something like:
`The database addons might not be free of cost and a paid plan is required to use them.`

**Comment by prokan468** [comment]
Hi, @ivanmonteiro I have made the changes. Please do review it again and let me know if there are any more changes to be made.

**Comment by nicolas63** [comment]
Looks good to me if @ivanmonteiro are ok i can merge

**Comment by ivanmonteiro** [comment]
> Looks good to me if @ivanmonteiro are ok i can merge

Looks good to me

**Comment by prokan468** [comment]
Thanks @ivanmonteiro and @nicolas63 . Appreciate it! Please do merge this PR.

**Comment by prokan468** [comment]
I am not sure as to why this is failing two checks, which it wasn't earlier. @nicolas63 Do I do something from my side or would that be taken care of by you?

**Comment by nicolas63** [comment]
It's not on your side, i will rerun the pipeline

**Comment by nicolas63** [comment]
Thanks for your contribution !

**Pull request merged**